test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py

```diff
@@ -0,0 +1,205 @@
+            )
+
+    with TestRun.step("Flush all dirty data."):
+        cache.flush_cache()
+
+    with TestRun.step("Change cleaning policy to ALRU and set parameters."):
+        cache.set_cleaning_policy(CleaningPolicy.alru)
+
+        params = FlushParametersAlru(
+            wake_up_time=time_wake_up,
+            staleness_time=time_idle,
+            flush_max_buffers=int(data_to_flush.get_value(Unit.Blocks4096)),
+            activity_threshold=time_idle
+        )
+        TestRun.LOGGER.info(str(params))
+        cache.set_params_alru(params)
+        new_params = cache.get_flush_parameters_alru()
+
+    with TestRun.step("Check if ALRU parameters are configured successfully."):
+        if cache.get_cleaning_policy() != CleaningPolicy.alru:
+            TestRun.fail("ALRU cleaning policy is not set!")
+        if new_params != params:
+            TestRun.fail("ALRU parameters are not changed.\n"
+                         f"Expected: {params}\nActual: {new_params}")
+
+    with TestRun.step("Fill core with dirty data."):
+        dd = (
+            Dd().input("/dev/zero")
+                .output(core.path)
+                .block_size(Size(1, Unit.MiB))
+                .oflag("direct")
+        )
+        dd.run()
+
+    with TestRun.step("Check core statistics."):
+        core_dirty_before = core.get_dirty_blocks()
+
+    with TestRun.step(f"Wait {int(wait_before_flush.total_seconds())} seconds "
+                      f"and check statistics again."):
+        # Wait until shortly before expected flushes
+        time.sleep(int(wait_before_flush.total_seconds()))
+        core_dirty_after = core.get_dirty_blocks()
+        if core_dirty_before != core_dirty_after:
+            TestRun.LOGGER.error(
+                f"Dirty data statistics differ before assumed cleaning thread launch.\n"
+                f"dirty data before pause: {core_dirty_before}\n"
+                f"dirty data after pause: {core_dirty_after}"
+            )
+
+    with TestRun.step(f"Wait {int((wait_after_flush - wait_before_flush).total_seconds())} seconds "
+                      f"and check statistics again."):
+        # Wait until flushes are certain to occur
+        time.sleep(int((wait_after_flush - wait_before_flush).total_seconds()))
+        core_dirty_after = core.get_dirty_blocks()
+        if core_dirty_before == core_dirty_after:
+            TestRun.LOGGER.error(
+                f"Dirty data statistics are the same despite the ALRU policy was used.\n"
+                f"dirty data before pause: {core_dirty_before}\n"
+                f"dirty data after pause: {core_dirty_after}"
+            )
+        elif core_dirty_before != core_dirty_after + data_to_flush:
+            TestRun.LOGGER.error(
+                f"Number of dirty blocks flushed differs from configured in policy.\n"
+                f"configured dirty data to flush: {data_to_flush}\n"
+                f"currently dirty data flushed: "
```

Comment:
Just a suggestion to modify those strings to "expected dirty data to flush" and "actual dirty data flushed" to fit the convention.